packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -781,27 +786,67 @@ export const ListBase = CollectionWidget.inherit({
     }
   },
 
-  _renderGroup(index, group) {
-    const $groupElement = $('<div>')
-      .addClass(LIST_GROUP_CLASS)
-      .appendTo(this._getItemsContainer());
+  _setGroupAria($group, groupHeaderId): void {
+    const { collapsibleGroups } = this.option();
 
-    const id = `dx-${new Guid().toString()}`;
     const groupAria = {
-      role: 'group',
+      role: collapsibleGroups ? undefined : 'group',
       // eslint-disable-next-line spellcheck/spell-checker
-      labelledby: id,
+      labelledby: collapsibleGroups ? undefined : groupHeaderId,
     };
 
-    this.setAria(groupAria, $groupElement);
+    this.setAria(groupAria, $group);
+  },
+
+  _updateGroupHeaderAriaExpanded($groupHeader, expanded): void {
+    this.setAria({ expanded }, $groupHeader);
+  },
+
+  _setGroupHeaderAria($groupHeader, listGroupBodyId): void {
+    const { collapsibleGroups } = this.option();
+
+    const groupHeaderAria = {
+      role: collapsibleGroups ? 'button' : undefined,
+      expanded: collapsibleGroups ? true : undefined,
+      controls: collapsibleGroups ? listGroupBodyId : undefined,
+    };
+
+    this.setAria(groupHeaderAria, $groupHeader);
+  },
+
+  _setGroupBodyAria($groupBody, groupHeaderId): void {
+    const { collapsibleGroups } = this.option();
+
+    const groupHeaderAria = {
+      role: collapsibleGroups ? 'listbox' : undefined,
+      // eslint-disable-next-line spellcheck/spell-checker
+      labelledby: collapsibleGroups ? groupHeaderId : undefined,
+    };
+
+    this.setAria(groupHeaderAria, $groupBody);
+  },
+
+  _renderGroup(index, group) {
+    const $groupElement = $('<div>')
+      .addClass(LIST_GROUP_CLASS)
+      .appendTo(this._getItemsContainer());
+
+    const groupHeaderId = `dx-${new Guid().toString()}`;
 
     const $groupHeaderElement = $('<div>')
       .addClass(LIST_GROUP_HEADER_CLASS)
-      .attr('id', id)
+      .attr('id', groupHeaderId)
       .appendTo($groupElement);
 
-    const groupTemplateName = this.option('groupTemplate');
-    const groupTemplate = this._getTemplate(group.template || groupTemplateName, group, index, $groupHeaderElement);
+    const { groupTemplate: templateName } = this.option();
+
+    const groupTemplate = this._getTemplate(
```

Comment:
JFYI: we have alias to get template by option name directly
https://github.com/DevExpress/DevExtreme/blob/46b526fbcbdcdb33a32fada7688ae46f6f0273ef/packages/devextreme/js/__internal/core/widget/dom_component.ts#L562-L563

Reply:
https://github.com/DevExpress/DevExtreme/blob/46b526fbcbdcdb33a32fada7688ae46f6f0273ef/packages/devextreme/js/__internal/core/widget/dom_component.ts#L562-L563